python/GangaAtlas/Lib/Athena/Athena.py

```diff
@@ -1004,6 +1004,13 @@ def prepare(self, force=False, **options):
 
         logger.info('Found Working Directory %s',self.userarea)
         logger.info('Found ATLAS Release %s',self.atlas_release)
+
+        # check for nightlies
+        if "AtlasBuildStamp" in os.environ:
+            # change production as the release found above is one ahead of the actual
+            logger.info('Found ATLAS Nightly release. Setting atlas_production appropriately')
+            self.atlas_production = os.environ['AtlasBuildBranch'] + ',' + os.environ['AtlasBuildStamp']
```

Comment:
`self.atlas_production` is assigned higher up in this file. As you assign rather than add here, is there a dager that something earlier is missed?